modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java

```diff
@@ -112,6 +112,26 @@ private boolean _isAPISchema(long apiSchemaId) throws Exception {
 		return true;
 	}
 
+	private boolean _isValidPathParameterString(String pathParameterString) {
```

Comment:
I think that it make sense!

Reply:
Hey @4lejandrito, this make sense, however what I'm trying to avoid is this use case when the user types these paths:
`/students/{{student}`
`/students/{{student}}`

This `_isValidPathParameterString` apart from taking into account the if the path parameter value is between curly braces it will also take care about how many times appears

Reply:
Okey, I found a way to do it by a regex form :grin: